tests/integration/data/tostac/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml

```diff
@@ -3,7 +3,7 @@
 $schema: https://schemas.opendatacube.org/dataset
 id: 2aa69fcf-aa55-4747-9d95-3652f9fe79b0
 
-label: ga_ls8c_ard_3-1-0_088080_2020-05-25_final
+label: ga_ls8c_ard_3-2-0_088080_2020-05-25_final
```

Comment:
Same with these two files in the `tostac` repo. Can leave them as-is. 

They're previously-packaged files, not data running through the new packager.

Reply:
And if we leave them as-is now, they wont show up on future find-replaces of similar versions :slightly_smiling_face: